src/MultilingualConfigOverride.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
```

Comment:
Missing `declare(strict_types = 1)`. 

Can you please add this and also the return types of all methods?